test/ng/compileSpec.js

```diff
@@ -4244,6 +4244,22 @@ describe('$compile', function() {
       if (!/chrome/i.test(navigator.userAgent)) return;
```

Comment:
unrelated to this issue, but can we change this to `if (!support.classes) return;` ?